examples/backtest/use_clock_timer/run_backtest.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+# -------------------------------------------------------------------------------------------------
+#  Copyright (C) 2015-2025 Nautech Systems Pty Ltd. All rights reserved.
+#  https://nautechsystems.io
+#
+#  Licensed under the GNU Lesser General Public License Version 3.0 (the "License");
+#  You may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at https://www.gnu.org/licenses/lgpl-3.0.en.html
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# -------------------------------------------------------------------------------------------------
+
+from decimal import Decimal
+
+import pandas as pd
+from strategy import SimpleTimerStrategy
+
+from nautilus_trader import TEST_DATA_DIR
+from nautilus_trader.backtest.engine import BacktestEngine
+from nautilus_trader.config import BacktestEngineConfig
+from nautilus_trader.config import LoggingConfig
+from nautilus_trader.model import TraderId
+from nautilus_trader.model.currencies import USD
+from nautilus_trader.model.data import Bar
+from nautilus_trader.model.data import BarType
+from nautilus_trader.model.enums import AccountType
+from nautilus_trader.model.enums import OmsType
+from nautilus_trader.model.identifiers import Venue
+from nautilus_trader.model.objects import Money
+from nautilus_trader.persistence.wranglers import BarDataWrangler
+from nautilus_trader.test_kit.providers import TestInstrumentProvider
+
+
+if __name__ == "__main__":
+    # Step 1: Configure and create backtest engine
+    engine_config = BacktestEngineConfig(
+        trader_id=TraderId("BACKTEST_TRADER-001"),
+        logging=LoggingConfig(
+            log_level="DEBUG",  # set DEBUG log level for console to see loaded bars in logs
+        ),
+    )
+    engine = BacktestEngine(config=engine_config)
+
+    # Step 2: Define exchange and add it to the engine
+    GLOBEX = Venue("GLBX")
```

Comment:
We're moving towards MIC codes for these venues (`XCME`). This is working for Databento except for continuous contracts.

Reply:
Implemented.